addons/xeh/fnc_initDisplay3DEN.sqf

```diff
@@ -8,14 +8,5 @@ add3DENEventHandler ["OnMissionPreviewEnd", {[] call CBA_fnc_preInit}];
 
 // switching terrains in 3den will reset missionNamespace
 add3DENEventHandler ["OnTerrainNew", {[] call CBA_fnc_preInit}];
-
-private _fnc_preInitMissionConfig = {
-    {
-        if (_x select 0 == "" && {_x select 1 == "preInit"}) then {
-            [] call (_x select 2);
-        };
-    } forEach (missionConfigFile call CBA_fnc_compileEventHandlers);
-};
-
-add3DENEventHandler ["OnMissionNew", _fnc_preInitMissionConfig];
-add3DENEventHandler ["OnMissionLoad", _fnc_preInitMissionConfig];
+add3DENEventHandler ["OnMissionNew", {[] call CBA_fnc_preInit}];
```

Comment:
OnTerrainNew is still needed as it fires after OnMissionLoad and the missionNamespace get's wiped inbetween them for whatever reason.